components/StarSearch/StarSearchCompactHeader.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import { Meta, StoryObj } from "@storybook/react";
+import { action } from "@storybook/addon-actions";
+import { StarSearchCompactHeader } from "./StarSearchCompactHeader";
+
+const meta: Meta<typeof StarSearchCompactHeader> = {
+  title: "Components/StarSearch/StarSearchCompactHeader",
+  component: StarSearchCompactHeader,
+  args: {
+    onBack: () => action("onBack")(),
+    onShare: () => action("onShare")(),
+    onNewChat: () => action("onNewChat")(),
+    onShowHistory: () => action("onShowHistory")(),
+    onClose: () => action("onClose")(),
+  },
+  decorators: [(story) => <div className="max-w-md ">{story()}</div>],
```

Comment:
I added a [Storybook decorator](https://storybook.js.org/docs/writing-stories/decorators) to container the compact header, but once this is in the embedded StarSearch component, it will expand to whatever width that has.